packages/dds/merge-tree/src/test/client.conflictFarm.spec.ts

```diff
@@ -78,14 +78,20 @@ function runConflictFarmTests(opts: IConflictFarmConfig, extraSeed?: number): vo
 				name: "with obliterate",
 				config: {
 					...opts,
+					generateSidedObliterates: true,
```

Comment:
i don't think this test needs a config for this, the trees can just have it on like regular obiliterate, and its up to the test if they generate it or not